framework/src/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -706,18 +705,18 @@ private static Receiver parseArray(String s, FlowExpressionContext context, Tree
         return result;
     }
 
+    // TODO: this returns true for "(a)+(b)" where the parens are not matching.
```

Comment:
I'm a bit confused about the `(a)+(b)` example and why the parens don't match. Should it be something more obviously non-matching like `(a + (b)`?